### PR TITLE
fix bugs in Move

### PR DESCRIPTION
**Pull request by songbinliu**

### CODE REVIEW

File: pkg/action/executor/rescheduler.go [hunk 1]
```diff
@@ -214,12 +214,12 @@ func (r *ReScheduler) reSchedule(action *turboaction.TurboAction) (*turboaction.
 	}
 
 	preScheduler, err := f(r.kubeClient, namespace, parentName, "", DefaultNoneExistSchedulerName)
+	defer f(r.kubeClient, namespace, parentName, DefaultNoneExistSchedulerName, preScheduler)
```
Comment (DongyiYang) [inline pkg/action/executor/rescheduler.go]: if there is err, do we still get correct value for preScheduler?

Comment (songbinliu) [inline pkg/action/executor/rescheduler.go]: yea, 
(1) If there is an  error, and fails to update the schedulerName, then preScheduler will be set to empty(""); in this case, the following call of f() in line-217 will do nothing.

(2) If there is an error, and the schedulerName  **may** have been updated, then the following call of f() in line-217 will try to restore the schedulerName.

Comment (songbinliu) [inline pkg/action/executor/rescheduler.go]: or this way may be better:

restore := func() {
		if preScheduler != "" {
			f(client, nameSpace, *noexistSchedulerName, preScheduler)
		}
	}
	defer restore()

Comment (DongyiYang) [inline pkg/action/executor/rescheduler.go]: OK, got it. I think a better way to write this is 
```go
preScheduler := getScheduler()
defer resetSchedler(preScheduler)
setScheduler(DefaultNoneExistSchedulerName)
```

But this way also works and I am fine with it.

**Pull request merged**